src/envoy/tcp/mixer/tcp_filter.cc

```diff
@@ -89,6 +94,7 @@ class TcpInstance : public Network::Filter,
 
   istio::mixerclient::CancelFunc cancel_check_;
   TcpMixerControl& mixer_control_;
+  const std::string uuid_;
```

Comment:
not to copy uuid for each connection,  maybe let mixer_control keep mixer_config reference, and directly use the one in config.